function_migrations.go

```diff
@@ -282,6 +283,32 @@ func migrateToSpecsStructure(f1 Function, m migration) (Function, error) {
 	return f1, nil
 }
 
+// migrateFromInvokeStructure migrates functions prior 0.35.0 via changing
+// the Invocation.format(string) to new Invoke(string) to minimalize func.yaml
+// file. When Invoke now holds default value (http) it will not show up in
+// func.yaml as the default value is implicitly expected. Otherwise if Invoke
+// is non-default value, it will be written in func.yaml.
+func migrateFromInvokeStructure(f1 Function, m migration) (Function, error) {
+	// Load the Function using pertinent parts of the previous version's schema:
+	f0Filename := filepath.Join(f1.Root, FunctionFile)
+	bb, err := os.ReadFile(f0Filename)
+	if err != nil {
+		return f1, errors.New("migration 'migrateFromInvokeStructure' error: " + err.Error())
+	}
+	f0 := migrateFromInvokeStructure_invocation{}
+	if err = yaml.Unmarshal(bb, &f0); err != nil {
+		return f1, errors.New("migration 'migrateFromInvokeStructure' error: " + err.Error())
+	}
+
+	if f0.Invocation.Format != "" && f0.Invocation.Format != DefaultInvocationFormat {
```

Comment:
I was wondering - Should i change this Default format to "locked in time" "http" string as well?